keras_hub/src/tests/test_case.py

```diff
@@ -141,7 +142,10 @@ def __init__(self, layer):
 
                 def call(self, x):
                     if isinstance(x, dict):
-                        return self.layer(**x)
```

Comment:
I don't think we should special case here - can we pass a nested dict as input to the layer?